src/preferences/preferences.hpp

```diff
@@ -817,6 +817,7 @@ class prefs
 			prefs_list::completed_campaigns,
 			prefs_list::history,
 			prefs_list::options,
+			prefs_list::server,
```

Comment:
There is also no `prefs_list::acquaintance` is that intentional?

Reply:
I'll need to double check - its handling is a bit different, but it should probably be brought into line with how everything else is done.

Reply:
So, yes, it was also missed. It's one of several preferences where it reads its values from the preferences config into a class/struct to represent them instead of directly using the preferences config, so that needs to stay as it is in the destructor.